Flake: TestClaimSchema

Lately I'm seeing TestClaimSchema fail just on CI. When I run it locally, it always passes (and has passed just fine previously).

Our dependency for jsonschema hasn't changed in a long time so I'm not sure what's causing the EOF.

```
=== RUN   TestClaimSchema
    claim_test.go:224: 
        	Error Trace:	claim_test.go:224
        	Error:      	Received unexpected error:
        	            	EOF
        	            	unable to compile schema validator for schema
        	            	
        	            	github.com/cnabio/cnab-go/schema.Validate
        	            		/home/vsts/work/1/s/schema/validation.go:50
        	            	github.com/cnabio/cnab-go/schema.ValidateClaim
        	            		/home/vsts/work/1/s/schema/validation.go:30
        	            	github.com/cnabio/cnab-go/claim.TestClaimSchema
        	            		/home/vsts/work/1/s/claim/claim_test.go:223
        	            	testing.tRunner
        	            		/opt/hostedtoolcache/go/1.17.8/x64/src/testing/testing.go:1259
        	            	runtime.goexit
        	            		/opt/hostedtoolcache/go/1.17.8/x64/src/runtime/asm_amd64.s:1581
        	Test:       	TestClaimSchema
        	Messages:   	failed to validate claim schema
--- FAIL: TestClaimSchema (0.00s)

```

--- a/fetch-schemas.go
+++ b/fetch-schemas.go
@@ -27,6 +27,8 @@ func main() {
 		bytes, err := fetchSchema(schema, version)
 		if err != nil {
 			fmt.Printf("unable to fetch %s schema with version %s: %s\n", schema, version, err.Error())
+			// if cdn.cnab.io is not reachable, we stick with default schema
+			return
 		}
 
 		err = writeSchema(schema, bytes)

--- a/fetch-schemas.go
+++ b/fetch-schemas.go
@@ -28,7 +28,7 @@ func main() {
 		if err != nil {
 			fmt.Printf("unable to fetch %s schema with version %s: %s\n", schema, version, err.Error())
 			// if cdn.cnab.io is not reachable, we stick with default schema
-			return
+			continue
 		}
 
 		err = writeSchema(schema, bytes)

